src/algebra/order/hom/monoid.lean

```diff
@@ -110,10 +118,15 @@ class order_monoid_hom_class (F : Type*) (α β : out_param $ Type*)
   extends monoid_hom_class F α β :=
 (monotone (f : F) : monotone f)
 
+attribute [to_additive] order_monoid_hom_class.to_monoid_hom_class
```

Comment:
AFAIR, the proper way to deal with old structure commands and `to_additive` is to add `@[ancestors]` to both structures.

Reply:
Thanks! Yes that seems to work.